### PR TITLE
Revert "Update CLI install guide"

### PR DESCRIPTION
**Pull request by wtrocki**
Reverts redhat-developer/app-services-guides#373 

After checking release process today I see that release involves number of manual steps (overall 3 hours of engineering time vs 1 minute upstream) and also requires jira tickets for teams that are not accessible for holiday period.
I'm really concerned about lack of ability to release in timely fashion over holiday period. 

This PR reverts changes we have made and restores upstream releases process for the short time (+2 sprints) the issues with downstream process will be resolved

### CODE REVIEW

**Comment by wtrocki** [comment]
@craicoverflow created as placeholder but waiting for approval.

**Comment by wtrocki** [comment]
Approved by PM. We can do that now

**Review by craicoverflow — APPROVED** [review]

**Comment by wtrocki** [comment]
@smccarthy-ie FYI - we would need to apply this patch downstream as well.

**Pull request merged**
